### ISSUE TITLE
delete several lines

### ISSUE THREAD

**Issue by hanxianlong** [XVimProject/XVim2#66]
When select several lines by mouse, and then press Key x to delete them, XVim2 doesn't perform as XVIM or VIM in IntelliJ IDEA.
What I want is to delete all my select lines, however in XVIM2 it just deselects the selected lines and deletes the first character in first line.
Is it possible to delete all selected lines by select them with mouse and delete them with pressing the key "x"?
 thanks.

**Comment by sobri909** [comment]
I think this is a more general problem of visual select with mouse being broken. It looks like the mouse selection isn't initiating command mode.

**Comment by hanxianlong** [comment]
@sobri909 I agree with you, but do you have any ideas about this problem to help improving it  :)
